packages/components/src/components/ContentCard/ContentCard.tsx

```diff
@@ -23,38 +23,26 @@ type CommonProps = {
   tag?: string;
   /** Sets the date to be rendered together with calendar icon */
   date?: string;
-  /** Sets the content button text */
-  buttonText?: string;
-  /** Sets a callback to content button */
-  onClickButton?: () => void;
   /** Sets the image position on the card */
   imagePosition?: ImagePosition;
   /** Sets an icon to be rendered over the image */
   iconUrl?: string;
   /** Sets the background color according with the type */
   background?: Background;
-};
-
-type InteractiveCard = {
-  /** Sets the URL to be added to the card when it behaves as a link */
-  href: string;
-  linkHref?: undefined;
-  linkText?: undefined;
-  as?: undefined;
-};
-
-type RegularCard = {
-  href?: undefined;
-  /** If the entire card is not a link this prop adds a link on the content body. This prop is mutually exclusive to "href" */
-  linkHref?: string;
-  /** Sets the text to the link on the content body */
-  linkText?: string;
+  /** Sets the type of the clickable element */
+  interactiveElementType?: "anchor" | "button" | "card";
+  /** Sets the href to be added to the interactive element */
+  href?: string;
+  /** Where to display the linked URL. Same as: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target */
+  target?: HTMLAnchorElement["target"];
+  /** Callback to be used when the interactive element is clicked */
+  onClick?: React.MouseEventHandler;
+  /** Sets the text to the interactive element when it's anchor or button */
+  ctaText?: string;
   /** Used by StyledComponents to render the component as a specific tag. If href is passed it'll be rendered as "a" */
   as?: React.ComponentProps<typeof Box.div>["as"];
```

Comment:
I'm not sure if we should keep this property because the developer doesn't have full control of it. And thinking about it, what would be the scenario to not use a `div` or a `a`?

Reply:
It will be used when the dev wants to create a listitem because we can list more than one card.